lab_6_pipeline/pipeline.py

```diff
@@ -239,11 +325,21 @@ def run(self) -> None:
         """
 
 
-def main() -> None:
+def main4() -> None:
+    corpus_manager = CorpusManager(path_to_raw_txt_data=ASSETS_PATH)
+    pipeline = TextProcessingPipeline(corpus_manager)
+    pipeline.run()
+
+
+def main6() -> None:
     """
     Entrypoint for pipeline module.
     """
+    corpus_manager = CorpusManager(path_to_raw_txt_data=ASSETS_PATH)
+    udpipe_analyzer = UDPipeAnalyzer()
+    pipeline = TextProcessingPipeline(corpus_manager, udpipe_analyzer)
+    pipeline.run()
```

Comment:
you do not need to have separate demonstrations. all marks should be demonstrated in single `main`

Reply:
Ok